src/common/config/state/runtime.js

```diff
@@ -14,6 +14,7 @@ const model = agentId => { return {
   customTransaction: undefined,
   disabled: false,
   features: {},
+  isolatedBacklog: false,
```

Comment:
isolatedBacklog is used by the EE in the MFE loader, to prevent EE context and buffers from bubbling up